src/arch/xtensa/CMakeLists.txt

```diff
@@ -538,7 +538,7 @@ add_custom_target(
 # .ri not signed: nothing variable in it to erase.
 set(UNSIGNED_RI         byt cht hsw bdw sue)
 # NXP
-list(APPEND UNSIGNED_RI imx8 imx8x imx8m)
+list(APPEND UNSIGNED_RI imx8 imx8x imx8m imx8ulp)
```

Comment:
```suggestion
if (${fw_name} MATCHES "imx8.*") # UNTESTED
  list(APPEND UNSIGNED_RI ${fw_name})
endif()
```

Reply:
@marc-hb i would prefer to keep the list explicit. Then moving just one member out of list will be easier. Also, adding new elements to the list might not respect the existing regular expression.

Reply:
I don't see why you would need to remove one member (any plan to use rimage signatures?) and I think regular expressions can be updated but it's your section, you're the boss :-) PR already approved.